src/encoded/static/components/search.js

```diff
@@ -1043,6 +1043,16 @@ export const FacetList = (props) => {
     // Combine facets from search results with special facets, and treat them mostly the same.
     const allFacets = SpecialFacetRegistry.Facet.getFacets().concat(facets);
 
+    const clearFiltersUrl = (contextObj) => {
+        if (!contextObj || !contextObj.clear_filters) {
+            return '';
+        }
+
+        const hasControlType = contextObj.facets.map((c) => c.field).includes('control_type');
+
+        return `${contextObj.clear_filters}${hasControlType ? '&control_type!=*' : ''}`;
+    };
```

Comment:
What might have been an interesting alternative is a new service that lets facets for certain “type=” values to include a list of query-string parameters not allowed to get cleared. That way if another case popped up, you could just add to the list instead of modifying this code.

Though frankly, I think the spec for this ticket is wrong. It should just allow clearing filters the usual way, and the facet should show an “either” for that case.